modules/apps/depot/depot-web/src/main/resources/META-INF/resources/js/ConnectedSiteDropdownPropsTransformer.js

```diff
@@ -25,7 +23,25 @@ class ConnectedSiteDropdownDefaultEventHandler extends DefaultEventHandler {
 		) {
 			submitForm(document.hrefFm, itemData.disconnectSiteActionURL);
 		}
-	}
-}
+	},
+};
+
+export default function propsTransformer({items, ...otherProps}) {
+	return {
+		...otherProps,
+		items: items.map((item) => {
+			return {
+				...item,
+				onClick(event) {
+					const action = item.data?.action;
 
-export default ConnectedSiteDropdownDefaultEventHandler;
+					if (action) {
+						event.preventDefault();
+
+						ACTIONS[action](item.data);
```

Comment:
Seems odd to have an `ACTIONS` object with exactly one key in it.

Reply:
I agree... I'll keep it for now, pending a deeper review of https://github.com/liferay-frontend/liferay-portal/pull/2#discussion_r448217443 so at least it stays consistent in the places we apply and is easier later to unify in something different.